module3/src/controllers/group.controller.ts

```diff
@@ -0,0 +1,63 @@
+import { GroupService } from '../services/group.service';
+import express from 'express';
+import { Error } from 'sequelize';
+import { validateSchema } from '../utils/utils';
+import { groupSchema } from '../validators/group';
+
+const groupService = new GroupService();
+
+export const groupsRouter = express.Router();
+groupsRouter.use(express.json());
+
+groupsRouter.get('/', (req, res) => {
+    groupService.getAllGroups()
+        .then((value: unknown) => {
+            res.json(value);
+        })
+        .catch((err: Error) => {
+            res.status(404).json(err);
+        });
+});
+
+groupsRouter.post('/', validateSchema(groupSchema), (req, res) => {
+    groupService.createGroup(req.body)
+        .then((result: unknown) => {
+            res.json(result);
+        })
+        .catch((err: Error) => {
+            res.status(404).json(err);
+        });
+});
+
+export const groupRouter = express.Router();
+groupRouter.use(express.json());
+
+groupRouter.get('/:id', (req, res) => {
+    groupService.getGroupById(req.params.id)
+        .then((result: unknown) => {
+            res.json(result);
+        })
+        .catch((err: Error) => {
+            res.status(404).json(err);
```

Comment:
you can use HttpStatuse from @types/node

Reply:
Added StatusCodes enum